scripts/hostcfgd

```diff
@@ -86,7 +88,17 @@ def signal_handler(sig, frame):
 
 def run_cmd(cmd, log_err=True, raise_exception=False):
     try:
-        subprocess.check_call(cmd, shell=True)
+        subprocess.check_call(cmd)
+    except Exception as err:
+        if log_err:
+            syslog.syslog(syslog.LOG_ERR, "{} - failed: return code - {}, output:\n{}"
+                  .format(err.cmd, err.returncode, err.output))
+        if raise_exception:
+            raise
+
+def run_cmd_pipe(cmd0, cmd1, cmd2, log_err=True, raise_exception=False):
```

Comment:
>run_cmd_pipe [](http://example.com/codeflow?start=4&length=12)

Similar function already implemented. Could you reuse in the scope of this repo?

Reply:
which function are you referring to?

Reply:
You implemented twice `run_commands_pipe` in this PR. I understand they are different a little bit. Could you try unify and reuse?

Reply:
Unified duplicate code to one function.